### PR TITLE
Add hakman and justinsb to etcd.io

### PR DESCRIPTION
**Pull request by hakman**
Ref: https://github.com/kubernetes/org/pull/5115#discussion_r1733653669

### CODE REVIEW

File: config/etcd-io/org.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ members:
 - fanminshi
 - fuweid
 - gryf
+- hakman
```
Comment (jmhbnz) [inline config/etcd-io/org.yaml:29]: Please also update `config/etcd-io/sig-etcd/teams.yaml` to add yourself back to the `etcd-operator-admins` team.

Comment (jmhbnz) [inline config/etcd-io/org.yaml:29]: Should we also add @justinsb  back via this pr as well?

**Review by jmhbnz — COMMENTED** [review]
Thanks @hakman 
/lgtm

cc @cblecker

**Review by cblecker — APPROVED** [review]

**Comment by hakman** [comment]
Thanks @cblecker @jmhbnz

**Pull request merged**